index.js

```diff
@@ -1 +1,41 @@
-console.log("Code your solution!")
+// Selectors
+
+// Event Listeners
+const parent = document.querySelector("form");
+parent.addEventListener("submit", (event) => {
+  event.preventDefault();
+  console.log(userInput);
+
+  const todoList = document.querySelector("ul");
+  const list = document.createElement("li");
```

Comment:
li elements are children of ul elements, however, you're not establishing their relationship. Below you only append the li element but never do anything with the ul.